content/docs/concurrent-mode-reference.md

```diff
@@ -98,7 +98,7 @@ Lorsque plusieurs composants ont besoin de récupérer des données, celles-ci p
   * `'collapsed'` affiche uniquement le repli du prochain élément dans la liste.
   * `'hidden'` n’affiche aucun élément non chargé.
 
-Remarquez que `SuspenseList` n’opère que sur les composants enfants `Suspense` et `SuspenseList` les plus proches d’elle. Elle ne recherche pas les frontières à plus d’un niveau de profondeur.  Ceci dit, il est possible d’imbriquer plusieurs composants `SuspenseList` pour construire des grilles.
+Remarquez que `SuspenseList` n’opère que sur les composants enfants `Suspense` et `SuspenseList` les plus proches d’elle. Elle ne recherche pas les périmètres à plus d’un niveau de profondeur.  Ceci dit, il est possible d’imbriquer plusieurs composants `SuspenseList` pour construire des grilles.
```

Comment:
`périmètres` précédement `frontières`, dans la doc originale est `boundaries`; finalement est ce que `limites` n'est pas plus approprié ?

Reply:
C'est par homogénéité avec les *périmètres d'erreurs* (*error boundaries*), pour lesquels *limites* n'est clairement pas idéal. Qui plus est avec Suspense on parle aussi vraiment d'un concept d'enrobage au sein de l'arbre, un périmètre transmet mieux cette notion de contenance que "limite"